lib/import/mobility.js

```diff
@@ -20,6 +20,24 @@ function yyyymmdd_to_iso(yyyymmdd) {
 }
 
 /**
+ * Return row whether row is valid
+ *
+ * @param{obj} obj - row from mobility file
+ * @return{bool}.
+ */
+function check_row_is_valid(obj) {
+  var missing = Object.keys(obj).find(
+    function(e) {return !obj[e];}
+  );
```

Comment:
Done.

I wanted to maintain an array of errors across all files, but wasn't sure how to it with the new promise sequence model we're using in import_mobility() in lib/import/mobility.js.